src/views/FieldReportForm/ResponseFields/styles.module.css

```
@@ -0,0 +1,2 @@
+.contacts-section {
+}
```

1. Let's not leave it empty
2. 
```suggestion
.response-fields {
}
```

---

Styling is a WIP